content/deep-dive/rpc/protobuf.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Protocol Buffers
+weight: 2
+---
+Interface definition languages, such as protobufs, are most commonly used to
+store and transmit data between applications.
+
+They define a way to *serialize* structures as text, as well as *de-serialize*

```

Comment:
Why hyphen? I noticed you're using the word "deserialize" directly somewhere above.